ghe-app-manifest.template.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ghe-app-for-jira",
+  "url": "https://github.com/marketplace/jira-software-github",
+  "redirect_url": "<<APP_HOST>>/github/manifest/<<UUID>>/complete",
+  "hook_attributes": {
+    "url": "<<APP_HOST>>/github/<<UUID>>/events"
+  },
+  "setup_url": "<<APP_HOST>>/github/<<UUID>>/setup",
+  "public": true,
+  "default_permissions": {
+    "actions": "read",
+    "security_events": "read",
+    "contents": "read",
+    "deployments": "read",
+    "issues": "write",
+    "metadata": "read",
+    "pull_requests":  "write",
+    "members": "read"
+  },
+  "default_events": [
+    "code_scanning_alert",
+    "commit_comment",
+    "create",
+    "delete",
+    "deployment_status",
+    "issue_comment",
+    "issues",
+    "pull_request",
+    "pull_request_review",
+    "push",
+    "repository",
+    "workflow_run"
+  ]
+}
```

Comment:
is there a reason we're not using handlebars to generate this JSON or even just using a function in the code with string interpolation?

Reply:
@mboudreau, updated with function with string interpolation